packages/components/src/public/files/FileAttachmentForm.tsx

```diff
@@ -18,18 +18,16 @@ import { Button } from 'react-bootstrap';
 import { List, Map } from 'immutable';
 import classNames from 'classnames';
 
-import { FormSection, Progress, LoadingSpinner, InferDomainResponse, inferDomainFromFile } from '../../..';
-
-import { FileAttachmentContainer } from './FileAttachmentContainer';
-import { FileGridPreviewProps, FilePreviewGrid } from './FilePreviewGrid';
+import { FileSizeLimitProps, FileGridPreviewProps } from './models';
+import { FileAttachmentContainer } from "../../internal/components/files/FileAttachmentContainer";
 import {
-    convertRowDataIntoPreviewData,
-    fileMatchesAcceptedFormat,
+    convertRowDataIntoPreviewData, fileMatchesAcceptedFormat,
     fileSizeLimitCompare,
-    getFileExtension,
-} from './actions';
-
-import { FileSizeLimitProps, SimpleResponse } from './models';
+    getFileExtension
+} from "../../internal/components/files/actions";
+import { FilePreviewGrid } from "../../internal/components/files/FilePreviewGrid";
+import { SimpleResponse } from "../../internal/components/files/models";
+import { FormSection, Progress, LoadingSpinner, InferDomainResponse, inferDomainFromFile } from "../..";
 
 interface FileAttachmentFormProps {
```

Comment:
The `FileAttachmentForm.spec.tsx` jest file above in the PR won't let me add a comment, so putting it here. Looks like the jest tests for that component use snapshot tests, so I assume you'll need to also move that file to this public location as well (i.e. `components/files/__snapshots__/FileAttachmentForm.spec.tsx.snap`).

Reply:
now moved